ara/templates/navbar.html

```diff
@@ -12,16 +12,6 @@
     <div class="navbar-collapse collapse">
       <ul class="nav navbar-nav">
         <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Hosts <span class="caret"></span></a>
```

Comment:
Yeah, I was thinking that in general we need to substantially re-architect the UI.  The filter-by-file stuff I put in is really just a stopgap measure; I'm sure there's a much nicer way to handle that, and of course we also want to include tools for exposing the file content now that we are collecting that.